### ISSUE TITLE
Clarify licensing

### ISSUE THREAD

**Issue by jamessan** [nh2/iso8601-time#7]
The only reference to the license under which this code is released is the `license: MIT` line in iso8601-time.cabal.

Unfortunately, `MIT` isn't actually a [clearly defined](https://en.wikipedia.org/wiki/MIT_License#Various_versions) license.  Would you mind adding an explicit declaration of license terms (typically a LICENSE or COPYING file at the top-level directory) to clarify this?

Cheers,
James

**Comment by nh2** [comment]
That's funny, the [Various Versions](https://en.wikipedia.org/wiki/MIT_License#Various_versions) section on Wikipedia didn't exist when I chose the license.

At the time I chose it, it was at [this version](https://en.wikipedia.org/w/index.php?title=MIT_License&oldid=646325537) when it referred solely to the `Expat License`.

(I chose MIT over BSD to not always have to explain whether or not I want the advertising clause, and now I discover somebody made a MIT variant with advetising clause so I'm back to the same problem...)

I mean the [MIT license as defined on the OSI's website](ttps://opensource.org/licenses/MIT). I will add this link to the README to make it unambiguous.
